simulator/dummy_perception_publisher/CMakeLists.txt

```diff
@@ -56,8 +56,14 @@ target_include_directories(dummy_perception_publisher_node
     $<INSTALL_INTERFACE:include>)
 
 # For using message definitions from the same package
-rosidl_target_interfaces(dummy_perception_publisher_node
-  ${PROJECT_NAME} "rosidl_typesupport_cpp")
+if(${rosidl_cmake_VERSION} VERSION_LESS 2.5.0)
+  rosidl_target_interfaces(dummy_perception_publisher_node
+    ${PROJECT_NAME} "rosidl_typesupport_cpp")
+else()
+  rosidl_get_typesupport_target(
+    cpp_typesupport_target ${PROJECT_NAME} "rosidl_typesupport_cpp")
+  target_link_libraries(dummy_perception_publisher_node "${cpp_typesupport_target}")
+endif()
```

Comment:
Could you link some information related to this issue?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/757#issue-1211442851